src/main/java/fi/otavanopisto/kuntaapi/server/integrations/mikkelinyt/MikkeliNytEntityUpdater.java

```diff
@@ -0,0 +1,286 @@
+
+    modificationHashCache.put(kuntaApiId.getId(), createPojoHash(attachment));
+    eventImageCache.put(new IdPair<EventId, AttachmentId>(eventId, kuntaApiId), attachment);
+  }
+
+  private Response<EventsResponse> listEvents(OrganizationId organizationId) {
+    String location = organizationSettingController.getSettingValue(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION);
+    String baseUrl = organizationSettingController.getSettingValue(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL);
+    String apiKey = getApiKey(organizationId);
+    
+    URI uri;
+    try {
+      URIBuilder uriBuilder = new URIBuilder(String.format("%s%s", baseUrl, "/json.php?showall=1"));
+
+      uriBuilder.addParameter("apiKey", apiKey);
+      if (StringUtils.isNotBlank(location)) {
+        uriBuilder.addParameter("location", location);
+      } else {
+        logger.warning("location not specified. Returning unfiltered event list");
+      }
+     
+      uri = uriBuilder.build();
+    } catch (URISyntaxException e) {
+      logger.log(Level.SEVERE, "Invalid uri", e);
+      return new Response<>(500, "Internal Server Error", null);
+    }
+    
+    return httpClient.doGETRequest(uri, new GenericHttpClient.ResultType<fi.otavanopisto.mikkelinyt.model.EventsResponse>() {});
+  }
+  
+  private String getApiKey(OrganizationId organizationId) {
+    return organizationSettingController.getSettingValue(organizationId, MikkeliNytConsts.ORGANIZATION_SETTING_APIKEY);
+  }
+  
+  private fi.otavanopisto.kuntaapi.server.rest.model.Event translate(EventId kuntaApiId, fi.otavanopisto.mikkelinyt.model.Event nytEvent) {
+    if (nytEvent == null) {
+      return null;
+    }
+    
+    fi.otavanopisto.kuntaapi.server.rest.model.Event result = new fi.otavanopisto.kuntaapi.server.rest.model.Event();
+    
+    result.setAddress(stripHtml(nytEvent.getAddress()));
+    result.setCity(stripHtml(nytEvent.getCity()));
+    result.setDescription(nytEvent.getDescription());
+    result.setEnd(parseOffsetDateTime(nytEvent.getEnd()));
+    result.setId(kuntaApiId.getId());
+    result.setName(stripHtml(nytEvent.getName()));
+    result.setOriginalUrl(nytEvent.getUrl());
+    result.setPlace(stripHtml(nytEvent.getPlace()));
+    result.setZip(stripHtml(nytEvent.getZip()));
+    result.setStart(parseOffsetDateTime(nytEvent.getStart()));
+    
+    return result;
+  }
+  
+  private Attachment translate(AttachmentId attachmentId, String imageUrl) {
+    DownloadMeta downloadMeta = binaryHttpClient.getDownloadMeta(imageUrl);
+    
+    Attachment attachment = new Attachment();
+    attachment.setId(attachmentId.getId());
+    
+    if (downloadMeta != null) {
+      attachment.setContentType(downloadMeta.getContentType());
+      attachment.setSize(downloadMeta.getSize() != null ? downloadMeta.getSize().longValue() : null);
+    }
+    
+    return attachment;
+  }
+  
+  private String stripHtml(String html) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "stripHtml" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)